internal/http/http.go

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) Mainflux
+// SPDX-License-Identifier: Apache-2.0
+
+package http
```

Comment:
Consider renaming to `httputil`. It just sounds like a better name for this package (also, no need for import aliases).